libs/designsystem/src/lib/scss/base/_variables.scss

```diff
@@ -51,6 +51,8 @@ $font-weight: (
   light: 300,
   normal: 400,
   bold: 700,
+  // black will be treated as a named css color if not quoted
+    'black': 900,
```

Comment:
Indentation seems to be off here

Reply:
Agree - seems to be a rule related to comments, as it matches with the line-weight map in the same file.